README.md

```diff
@@ -38,6 +39,23 @@ python3.5 -m venv env
 pip3 install -r requirements.txt
 python setup.py develop
 ```
+Also, a [Docker image](Dockerfile) is used to ship dcos-launch instead of a binary.
+Note: dcos-launch requires the use of local file paths, so you will need to volume mount:
+```
+docker run -it -v /my/home/dir:/dcos-launch create
```

Comment:
this overwrites the `/dcos-launch` dir removing capability (unless you are in the dcos-launch project directory locally).   It would be best to have `/dcos-launch` in the path and have a working directory which has mount access to a config file.